packages/cli/src/lib/decorators/preconditions/authenticated.ts

```diff
@@ -1,11 +1,23 @@
 import Command from '@oclif/command';
+import {Config} from '../../config/config';
 import {
   AuthenticationStatus,
   getAuthenticationStatus,
 } from '../../platform/authenticatedClient';
 
 export function IsAuthenticated() {
   return async function (target: Command) {
+    const cfg = new Config(global.config.configDir);
```

Comment:
Why this change ? 

It changes the meaning of the function: It's not about being "authenticated" in the platform anymore (as in: does the user have a valid access token as far as the platform is concerned), it's about verifying some value in a local configuration file.

If you really need to verify that "the user needs to have value X in his configuration file", then I would suggest creating a precondition function that does that, and to add the precondition it to all the commands that need it.